engine/vcs/gitlab/client_pull_request.go

```diff
@@ -23,5 +50,39 @@ func (c *gitlabClient) PullRequestComment(context.Context, string, sdk.VCSPullRe
 
 // PullRequestCreate create a new pullrequest
 func (c *gitlabClient) PullRequestCreate(ctx context.Context, repo string, pr sdk.VCSPullRequest) (sdk.VCSPullRequest, error) {
-	return sdk.VCSPullRequest{}, fmt.Errorf("not yet implemented")
+	mr, _, err := c.client.MergeRequests.CreateMergeRequest(repo, &gitlab.CreateMergeRequestOptions{
```

Comment:
it's the user that link the project to gitlab who create the pullrequest? is it possible to grant another user (like other vcs ) ?

Reply:
Will do in another PR